server/src/main/java/com/distkv/dst/server/runtime/workerpool/WorkerPool.java

```diff
@@ -13,14 +15,17 @@
 
   private boolean isMaster;
 
+  private List<String> slavers;
+
   private final ImmutableList<Worker> workers;
 
-  public WorkerPool(int shardNum, boolean isMaster) {
+  public WorkerPool(int shardNum, boolean isMaster, List<String> slavers) {
     this.shardNum = shardNum;
     this.isMaster = isMaster;
+    this.slavers = slavers;
     ImmutableList.Builder<Worker> builder = new ImmutableList.Builder<>();
     for (int i = 0; i < shardNum; ++i) {
-      Worker worker = new Worker();
+      Worker worker = new Worker(isMaster, slavers);
```

Comment:
I don't know whether we should create the clients for either worker pool or worker.

Any thoughts?

Reply:
I see. I think every worker can reuse a client

Reply:
Because a worker will not be robbed of resources

Reply:
It seems reasonable to put the clients in DstRuntime.

Reply:
ok